dash/callbacks.py

```diff
@@ -0,0 +1,132 @@
+from dash import Input, Output, callback, html, dcc
+from visualisations import create_visualisation
+from SQLConnection import SQLConnection
+from datetime import datetime
+
+# SQL connection variable
+# sql = SQLConnection('./ec2-dash/dash.db')
+
+# Update current time
+@callback(
+	Output(
+		"current_date",'children'
+	),
+	Output(
+		"current_time",'children'
+	),
+	[
+		Input(
+			'interval_component', 'n_intervals'
+		)
+	]
+)
+def update_current_time(n):
+	return datetime.now().strftime("%d/%m/%Y"),datetime.now().strftime("%H:%M")
+
+# Update page label button
+@callback(
+	Output(
+		'view_switch', 'children'
+	),
+	Output(
+		'page_link', 'href'
+	),
+	[
+		Input(
+			'view_switch', 'n_clicks'
+		)
+	]
+)
+def change_link(n):
+	print(n)
+	if n % 2 == 0:
+		return 'CURRENT', '/recent'
+	else:
+		return 'RECENT', '/'
+
+# # Update user info at the start of a new ride
```

Comment:
For final product, might be worth removing commented out code but I take it you may use it later or were experimenting

Reply:
Yeh agreed, we can tidy up the whole repo so we only merge with main with the files in staging that are necessary

Reply:
Yeah thats fine, this code is just stuff that I need/will be used, but was giving me errors because it wasn't connect to the db currently